util/util.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2018 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements utility functionalities to all OpenSDS northbound service.
+*/
+
+package util
+
+import (
+	"errors"
+	"fmt"
+	"github.com/opensds/opensds/client"
+	"github.com/opensds/opensds/pkg/model"
+)
+
+// GetHostIdByHostName returns hostid for given hostname if found
+func GetHostByHostName(client *client.Client, hostName string) (*model.HostSpec, error) {
+	hostLists, err := client.HostMgr.ListHosts()
+	if nil != err {
+		return nil, errors.New("Failed to list hosts")
+	}
+
+	for _, elem := range hostLists {
```

Comment:
Do we need to parse through all host list, Hope the API takes filter "?name=<hostName>"

Reply:
To be fixed. Raised an issue to track (https://github.com/opensds/nbp/issues/296)